presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -827,6 +827,8 @@ public Plan createPlan(Session session, @Language("SQL") String sql, List<PlanOp
                 sqlParser,
                 statsCalculator,
                 costCalculator,
+                nodeManager,
+                new NodeSchedulerConfig().setIncludeCoordinator(true),
```

Comment:
This config is being created twice in this class. Create in in the `LocalQueryRunner` constructor and store it as a field, to make sure it is always the same.